game/templates/game/board.html

```
@@ -196,6 +196,12 @@
                 <div class="captured-list" id="blackCaptured"></div>
                 <div class="captured-score">Points: <span id="blackPoints">+0</span></div>
             </div>
+            <div style="margin-top: 10px;">
+                <div style="display: flex; flex-direction: column; gap: 4px;">
+                    <input type="text" id="manualMoveInput" placeholder="Move (e.g. e2e4)" style="padding: 10px; border-radius: 5px; border: 1px solid #444; background: #1a1a2e; color: #fff; font-size: 0.9rem;" maxlength="5">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)